### DISCUSSION TITLE
Cannot find module 'pinia/dist/pinia.mjs'

### DISCUSSION THREAD

**Discussion by suityou01** [vuejs/pinia#2721] [Help and Questions] [answered]
### Reproduction

Reproduction

### Steps to reproduce the bug

Install nuxt
Install pinia using the instructions on https://pinia.vuejs.org/ssr/nuxt.html
Try to start nuxt
Get error ERROR  Cannot start nuxt:  Cannot find module 'pinia/dist/pinia.mjs'

### Expected behavior

Nuxt starts

### Actual behavior

Nuxt doesn't start

**Comment by Janmuixi** [comment, marked as answer]
I couldn't reproduce the error, I followed these steps:

· npx nuxi@latest init <project-name>
· added the following on package.json
`"overrides": {
  "vue": "latest"
}`
· npm install pinia @pinia/nuxt
· npm run dev

**Comment by lmc3s3f0r** [comment]
I'm having the same problem. The documentation here is wrong: https://pinia.vuejs.org/ssr/nuxt.html

It says you only need to run "npx nuxi@latest module add pinia", and that doesn't work.
